great_expectations/expectations/core/expect_column_distinct_values_to_be_in_set.py

```diff
@@ -141,50 +143,51 @@ def _atomic_prescriptive_template(
         configuration: Optional[ExpectationConfiguration] = None,
         result: Optional[ExpectationValidationResult] = None,
         runtime_configuration: Optional[dict] = None,
-        **kwargs,
-    ):
-        runtime_configuration = runtime_configuration or {}
-        include_column_name = (
-            False if runtime_configuration.get("include_column_name") is False else True
+    ) -> Tuple[str, dict, Union[dict, None]]:
+        renderer_configuration = RendererConfiguration(
+            configuration=configuration,
+            result=result,
+            runtime_configuration=runtime_configuration,
         )
-        styling = runtime_configuration.get("styling")
-
-        params = substitute_none_for_missing(
-            configuration.kwargs,
-            ["column", "value_set", "row_condition", "condition_parser"],
+        renderer_configuration.add_param(
+            name="column",
+            schema_type="string",
+        )
+        renderer_configuration.add_param(
+            name="value_set",
+            schema_type="array",
+        )
+        renderer_configuration.add_param(
+            name="row_condition",
+            schema_type="string",
+        )
+        renderer_configuration.add_param(
+            name="condition_parser",
+            schema_type="string",
         )
-        params_with_json_schema = {
-            "column": {"schema": {"type": "string"}, "value": params.get("column")},
-            "value_set": {
-                "schema": {"type": "array"},
-                "value": params.get("value_set"),
-            },
-            "row_condition": {
-                "schema": {"type": "string"},
-                "value": params.get("row_condition"),
-            },
-            "condition_parser": {
-                "schema": {"type": "string"},
-                "value": params.get("condition_parser"),
-            },
-        }
 
-        if params["value_set"] is None or len(params["value_set"]) == 0:
+        params: RendererParams
+        params = renderer_configuration.params
 
-            if include_column_name:
+        if not params.value_set or len(params.value_set.value) == 0:
```

Comment:
`RendererConfiguration.add_param()` will always add a new attribute to `RendererConfiguration.params`. In order to make the truthiness of a given attribute meaningful (not always `True`), we overload the `__bool__` and check the truthiness of `.value`. If `params.value_set.value` is falsy or 0 this conditional would evaluate to `True`. The `.value` could be appended to `params.value_set` and the behavior would remain the same.